server/src/test/java/com/saralein/server/FileHelperTest.java

```diff
@@ -56,11 +58,14 @@ public void getsMimeTypesOfFiles() {
     }
 
     @Test
-    public void getsNamesOfFilesInDirectory() {
-        try {
-            assertEquals(fileNames, fileHelper.listFileNames(root));
-        } catch (IOException e) {
-            fail("Failed to list file names in File Helper test.");
-        }
+    public void getsLengthOfFiles() throws IOException {
+        assertEquals(10, fileHelper.getFileLength(txtPath));
+        assertEquals(15587, fileHelper.getFileLength(jpgPath));
+        assertEquals(2628712, fileHelper.getFileLength(gifPath));
```

Comment:
Where do these numbers come from?

Reply:
Well, the txt file from counting the characters.  But the others are just hanging out in OS info.

Reply:
Is there a way to get those number programmatically?

Reply:
As @zacharyjdavy pointed out to me, it is probably not a good idea to get these programmatically. Maybe just using a variable instead of the literal?